dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/unternehmensfuehrungGovernance/sonstige/GdvUnternehmensfuehrungGovernanceSonstige.kt

```diff
@@ -0,0 +1,13 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.unternehmensfuehrungGovernance.sonstige
+
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+
+/**
+ * The data-model for the Sonstige section
+ */
+data class GdvUnternehmensfuehrungGovernanceSonstige(
+    val wirtschaftspruefer: String?,
+    val ceoVorsitzender: YesNo?,
```

Comment:
What exactly is meant by this? Is the question if the CEO is the Vorsitzender of this Governance ?

Reply:
Tooltip in the data dictionary says "Hat sich das Unternehmen im aktuellen Jahr der Berichterstattung von CEO/Vorsitzenden getrennt?"

Reply:
ok, but then the naming is rather strange, but probably due to the business side. is there a way for us to give feedback, because i never would have thought about this explanation when looking at the name of the field

Reply:
thank you for fixing this